### PR TITLE
Add release page

### PR DESCRIPTION
**Pull request by JonasHelming**
Signed-off-by: Jonas Helming <jhelming@eclipsesource.com>

### CODE REVIEW

**Review by vince-fugnitto — COMMENTED** [review]
@JonasHelming are that many changes to the `package-lock.json` intentional?

> 23,691 additions, 27 deletions.

---

I see, the old `package-lock.json` was created with an outdated version of `npm` and a one-time update was necessary:

```
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
```

**Comment by JonasHelming** [comment]
> @JonasHelming are that many changes to the `package-lock.json` intentional?
> 
> > 23,691 additions, 27 deletions.
> 
> I see, the old `package-lock.json` was created with an outdated version of `npm` and a one-time update was necessary:
> 
> ```
> npm WARN old lockfile
> npm WARN old lockfile The package-lock.json file was created with an old version of npm,
> npm WARN old lockfile so supplemental metadata must be fetched from the registry.
> npm WARN old lockfile
> npm WARN old lockfile This is a one-time fix-up, please be patient...
> npm WARN old lockfile
> ```

Yes, exactly, it needed an update since quite a while now

File: src/components/Releases.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/********************************************************************************
+ * Copyright (C) 2021 EclipseSource and others.
```
Comment (vince-fugnitto) [inline src/components/Releases.js:2]: minor: should it not be 2022?

```suggestion
 * Copyright (C) 2022 EclipseSource and others.
```

File: src/components/Releases.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+const Releases = () => (
+    <StyledResources>
+        <header className='header' role="banner">
+            <div className="row">
+                <Nav shouldRenderLogo={true} />
+                <div className="header__text-box">
+                    <h1 className="heading-primary">
+                        Releases
+                    </h1>
+                    <p>Eclipse Theia follows a monthly release cycle. You can find all releases including change logs and breaking changes <a href="https://github.com/eclipse-theia/theia/releases">here</a></p>
```
Comment (vince-fugnitto) [inline src/components/Releases.js:108]:
```suggestion
                    <p>Eclipse Theia follows a monthly release cycle. You can find all releases including changelogs and breaking changes <a href="https://github.com/eclipse-theia/theia/releases">here</a></p>
```

File: src/components/Releases.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                    <br></br>
+                    <p>In addition, Theia provides a "community release" every three month with extra focus on stability and compatibility. The list below shows all community releases including the next planned. In addition, you find technologies that integrate with Theia including the respective version that is supposed to be compatible with the community release. This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a> to be listed.</p>
```
Comment (vince-fugnitto) [inline src/components/Releases.js:110]: minor: I'd break this `<p>` into multiple lines, just so it's easier to read and update.

Comment (vince-fugnitto) [inline src/components/Releases.js:110]:
```suggestion
                    <p>In addition, Theia provides a "community release" every three month with extra focus on stability and compatibility. The list below shows all community releases including the next one planned. In addition, you can find technologies that integrate with Theia, including the respective version that is supposed to be compatible with the community release. This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report them at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a> to be listed.</p>
```

File: src/components/Releases.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+                </div>
+            </div>
+        </header>
+        {communityReleases.map(({ name, frameworks, releasedate, releasecandidatedate }, i) => {
+            return (
+                <section key={i} className="category row" id="frameworks">
+                    <h2 className="heading-secondary">{name}</h2>
+                    <p>Release Date: {releasedate}</p>
+                    <p>Release Candidate Date: {releasecandidatedate}</p>
```
Comment (vince-fugnitto) [inline src/components/Releases.js:120]: minor: for newcomers do we want to mention this is the "community" release?

```suggestion
                    <p>Community Release Candidate Date: {releasecandidatedate}</p>
```

**Review by vince-fugnitto — COMMENTED** [review]
I think we should probably improve the styling, most notably for the "Compatible Technologies" section which seems to take up more space than the actual community release itself. Given new technologies will be added (upon request) we should probably exclude logos for the time being or altogether.

image

File: src/components/Releases.js [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+                    <p>Community Release Candidate Date: {releasecandidatedate}</p>
+                    <br></br>
+                    <p><h3 className="heading-tertiary"><b>Compatible Technologies:</b></h3></p>
+                    {frameworks.map(
+                        ({ title, url, version, icon }, i) => {
+                            return (
+                                <div key={`${i}_${title}`} className="framework">
+                                            <h3 className="heading-tertiary">
+                                                <a href={url}>{title}, Version: {version}</a>
+                                            </h3>
```
Comment (vince-fugnitto) [inline src/components/Releases.js:122]: minor: should it be `h4` as "Compatible Technologies" (the title) already uses `h3` ?

File: src/components/Releases.js [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+                    <h1 className="heading-primary">
+                        Releases
+                    </h1>
+                    <p>Eclipse Theia follows a monthly release cycle. You can find all releases including changelogs and breaking changes <a href="https://github.com/eclipse-theia/theia/releases">here</a></p>
+                    <br></br>
+                    <p>In addition, Theia provides a "community release" every three month with extra focus on stability and compatibility.</p>
+                    <p>The list below shows all community releases including the next one planned. In addition, you can find technologies that integrate with Theia, including the respective version that is supposed to be compatible with the community release.</p>
+                    <p>This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report them at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a> to be listed.</p>
```
Comment (paul-marechal) [inline src/components/Releases.js:103]: Repetition?

```suggestion
                    <p>This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report them at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a>.</p>
```

**Pull request merged**